astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
I disagree--that's not really what the purpose of this function was.  If someone needs to actually test that a module can be imported the current solution is to use an explicit import statement.

At some point I think there would still be value in having some sort of optional dependency manager a la #63.  The scope of *this* function is intentionally very limited.